webpack/assets/javascripts/react_app/routes/common/PageLayout/PageLayout.js

```diff
@@ -50,16 +50,15 @@ const PageLayout = ({
           <Col className="title_filter" md={searchable ? 6 : 4}>
             {searchable && (
               <div id="search-bar">
-                {
-                  <SearchBar
-                    data={searchProps}
-                    initialQuery={searchQuery}
-                    onSearch={onSearch}
-                    onBookmarkClick={onBookmarkClick}
-                  />
-                }
+                <SearchBar
+                  data={searchProps}
+                  initialQuery={searchQuery}
+                  onSearch={onSearch}
+                  onBookmarkClick={onBookmarkClick}
+                />
               </div>
             )}
+            &nbsp;
```

Comment:
why is this needed?

Reply:
To be consistent with erb layout and have the same spacing between search bar and table.

https://github.com/theforeman/foreman/pull/6818#discussion_r341311263